jazkarta/shop/api.py

```diff
@@ -0,0 +1,14 @@
+from dateutil import parser
+from jazkarta.shop import storage
+
+
+def get_order_from_id(id):
+    """ An order_id is a string consisting of the userid and the
+    order date in iso formatr, joined by the pipe character. """
+    userid, datestr = id.split('|')
+    date = parser.parse(datestr)
+    if userid == 'orders':
```

Comment:
This makes me a bit uneasy since there's nothing to prevent there being a user whose user id is literally "orders".